graf2d/gpadv7/inc/ROOT/TObjectDisplayItem.hxx

```diff
@@ -61,10 +63,16 @@ public:
       if (fOwner) delete fObject;
    }
 
-   void AddTColor(int color_indx, const std::string &color_value)
+   void AddColor(int color_indx, const std::string &color_value)
    {
-      fColIndex.emplace_back(color_indx);
-      fColValue.emplace_back(color_value);
+      auto pos = std::find(fColIndex.begin(), fColIndex.end(), color_indx);
```

Comment:
Why isn't `fColIndex`, `fColValue` a `map<int /*idx*/, string /*value*/>`?

Reply:
Just to make it easier to handle. std::map has too large footprint in JSON